### PR TITLE
fix: version-bump generator removes beachball disallowedChangeType config

### LINKED ISSUE #25116: Beachball disallowed changetypes breaks nightly releases
Since nightly releases rely on prerelease versions, beachball's disallowed changetypes results in no releases from the nightly pipeline

Also beachball does not exit with an error when it encounters disallowed change types

image

### PR DESCRIPTION
**Pull request by ling1726**
Fixes #25116

### CODE REVIEW

File: tools/generators/version-bump/index.ts [hunk 1]
```diff
@@ -33,6 +33,11 @@ function runMigrationOnProject(host: Tree, schema: ValidatedSchema, userLog: Use
 
   updateJson(host, packageJsonPath, (packageJson: PackageJson) => {
     nextVersion = bumpVersion(packageJson, schema.bumpType, schema.prereleaseTag);
+
+    // nightly releases should bypass beachball disallowed changetypes
+    if (schema.bumpType === 'nightly' && packageJson.beachball?.disallowedChangeTypes) {
+      packageJson.beachball.disallowedChangeTypes = undefined;
```
Comment (Hotell) [inline tools/generators/version-bump/index.ts:39]: lets get rid of it completely

```suggestion
delete packageJson.beachball.disallowedChangeTypes
```

File: tools/types.ts [hunk 2]
```diff
@@ -31,6 +31,9 @@ export interface PackageJson {
   devDependencies?: Record<string, string>;
   peerDependencies?: Record<string, string>;
   exports?: Record<string, string | Partial<{ types: string; import: string; require: string }>>;
+  beachball?: {
```
Comment (Hotell) [inline tools/types.ts:34]: **suggestion:** can we extract these tool specific to separate interface and extend PackageJson ? I mean "beachball" is definitely not a standard package.json property

**Review by Hotell — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/Avatar-mount_base.html">1609</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/Avatar-mount.html">1619</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/Button-mount_base.html">1147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/Button-mount.html">1143</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/FluentProvider-mount_base.html">1911</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/FluentProvider-mount.html">1902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/FluentProviderWithTheme-mount_base.html">715</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/FluentProviderWithTheme-mount.html">710</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">666</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">665</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">703</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">702</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/MakeStyles-mount_base.html">2253</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/MakeStyles-mount.html">2259</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/SpinButton-mount_base.html">3185</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test-react-components/SpinButton-mount.html">3209</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/BaseButton-mount_base.html">1592</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/BaseButton-mount.html">1581</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Breadcrumb-mount_base.html">3832</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Breadcrumb-mount.html">3720</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Checkbox-mount_base.html">3474</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Checkbox-mount.html">3569</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/CheckboxBase-mount_base.html">3104</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/CheckboxBase-mount.html">3153</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ChoiceGroup-mount_base.html">6014</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ChoiceGroup-mount.html">6021</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ComboBox-mount_base.html">1575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ComboBox-mount.html">1654</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/CommandBar-mount_base.html">12573</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/CommandBar-mount.html">12488</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ContextualMenu-mount_base.html">14656</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ContextualMenu-mount.html">14616</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DefaultButton-mount_base.html">1876</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DefaultButton-mount.html">1844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DetailsRow-mount_base.html">4916</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DetailsRow-mount.html">4887</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DetailsRowFast-mount_base.html">4884</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DetailsRowFast-mount.html">4919</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DetailsRowNoStyles-mount_base.html">4610</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DetailsRowNoStyles-mount.html">4663</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Dialog-mount_base.html">4014</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Dialog-mount.html">4027</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DocumentCardTitle-mount_base.html">733</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/DocumentCardTitle-mount.html">751</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Dropdown-mount_base.html">4370</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Dropdown-mount.html">4379</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/FocusTrapZone-mount_base.html">2609</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/FocusTrapZone-mount.html">2697</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/FocusZone-mount_base.html">2449</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/FocusZone-mount.html">2548</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedList-mount_base.html">73034</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedList-mount.html">84286</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedList-virtual-rerender_base.html">33963</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedList-virtual-rerender.html">34041</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">115591</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedList-virtual-rerender-with-unmount.html">114657</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedListV2-mount_base.html">753</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedListV2-mount.html">702</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedListV2-virtual-rerender_base.html">693</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedListV2-virtual-rerender.html">680</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">701</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">723</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/IconButton-mount_base.html">2556</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/IconButton-mount.html">2594</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Label-mount_base.html">952</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Label-mount.html">934</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Layer-mount_base.html">5616</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Layer-mount.html">5609</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Link-mount_base.html">1056</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Link-mount.html">1120</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/MenuButton-mount_base.html">2301</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/MenuButton-mount.html">2300</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/MessageBar-mount_base.html">2965</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/MessageBar-mount.html">2985</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Nav-mount_base.html">4448</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Nav-mount.html">4400</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/OverflowSet-mount_base.html">1747</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/OverflowSet-mount.html">1764</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Panel-mount_base.html">3302</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Panel-mount.html">3210</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Persona-mount_base.html">1668</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Persona-mount.html">1669</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Pivot-mount_base.html">2158</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Pivot-mount.html">2147</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/PrimaryButton-mount_base.html">2048</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/PrimaryButton-mount.html">2028</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Rating-mount_base.html">9817</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Rating-mount.html">9742</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/SearchBox-mount_base.html">2119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/SearchBox-mount.html">2084</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Shimmer-mount_base.html">3877</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Shimmer-mount.html">3926</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Slider-mount_base.html">2770</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Slider-mount.html">2798</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/SpinButton-mount_base.html">6245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/SpinButton-mount.html">6310</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Spinner-mount_base.html">1026</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Spinner-mount.html">1027</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/SplitButton-mount_base.html">4158</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/SplitButton-mount.html">4174</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Stack-mount_base.html">1130</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Stack-mount.html">1134</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/StackWithIntrinsicChildren-mount_base.html">3295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/StackWithIntrinsicChildren-mount.html">3293</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/StackWithTextChildren-mount_base.html">6807</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/StackWithTextChildren-mount.html">6715</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/SwatchColorPicker-mount_base.html">14024</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/SwatchColorPicker-mount.html">14101</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/TagPicker-mount_base.html">3486</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/TagPicker-mount.html">3788</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/TeachingBubble-mount_base.html">114790</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/TeachingBubble-mount.html">116050</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Text-mount_base.html">1050</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Text-mount.html">1046</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/TextField-mount_base.html">2146</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/TextField-mount.html">2161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ThemeProvider-mount_base.html">1961</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ThemeProvider-mount.html">2010</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ThemeProvider-virtual-rerender_base.html">1398</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ThemeProvider-virtual-rerender.html">1369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2819</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2799</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Toggle-mount_base.html">1488</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/Toggle-mount.html">1473</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/buttonNative-mount_base.html">676</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25130/perf-test/buttonNative-mount.html">695</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Pull request merged**